nearuplib/nodelib.py

```diff
@@ -238,17 +256,31 @@ def get_genesis_md5sum(net):
     return download_near_s3(f'nearcore-deploy/{net}/genesis_md5sum').strip()
 
 
-def print_staking_key(home_dir):
+def amount_need_for_stake(chain_id):
+    if chain_id == 'betanet':
+        return 50000
+    else:
+        # a very high amount to prevent user from staking
+        return 50000000
+
+
+def print_staking_key(home_dir, chain_id):
     key_path = os.path.join(home_dir, 'validator_key.json')
     if not os.path.exists(key_path):
         return
 
     key_file = json.loads(open(key_path).read())
-    if not key_file['account_id']:
+    account_id = key_file['account_id']
+    pubkey = key_file['public_key']
+    if not account_id:
         print("Node is not staking. Re-run init to specify staking account.")
         return
-    print("Stake for user '%s' with '%s'" %
-          (key_file['account_id'], key_file['public_key']))
+    if chain_id in ['devnet', 'betanet', 'testnet']:
+        print(f"""Please staking with near-shell (https://github.com/near/near-shell):
+
+    near login --nodeUrl https://rpc.{chain_id}.near.org --networkId {chain_id} --accountId {account_id} --keyPath {key_path} --walletUrl https://wallet.{chain_id}.near.org --helperUrl https://helper.{chain_id}.near.org
+    near stake {account_id} {pubkey} {amount_need_for_stake(chain_id)} --nodeUrl https://rpc.{chain_id}.near.org --networkId {chain_id} --accountId {account_id} --keyPath {key_path} --walletUrl https://wallet.{chain_id}.near.org --helperUrl https://helper.{chain_id}.near.org # or above for staking on {chain_id}
```

Comment:
@vgrichina is there easier way to specify all urls to use betanet? For a freshly installed `npm install -g near-shell`